python_modules/dagster/dagster/_core/definitions/asset_graph_differ.py

```diff
@@ -6,11 +6,13 @@
 from dagster._core.errors import DagsterInvariantViolationError
 from dagster._core.remote_representation import ExternalRepository
 from dagster._core.workspace.context import BaseWorkspaceRequestContext
+from dagster._serdes import whitelist_for_serdes
 
 if TYPE_CHECKING:
     from dagster._core.definitions.events import AssetKey
 
 
+@whitelist_for_serdes
 class ChangeReason(Enum):
```

Comment:
`AssetDefinitionChangeKind` perhaps? `ChangeReason` feels very broad and not sure that "Reason" is the right word here.